examples/iam-policy-management.v1.test.js

```diff
@@ -65,6 +65,7 @@ describe('IamPolicyManagementV1', () => {
   let exampleAssignmentPolicyId;
   let exampleTargetAccountId;
   let exampleAssignmentETag;
+  let exampleAccountSettingsETag;
```

Comment:
If the API definition was outfitted with the link objects to define various inter-operation dependencies, then variables like these `exampleXXX` variables would be generated automatically and initialized and referenced in the proper places in the generated examples code.
Here is a tutorial video that shows how to do this:
https://secure.video.ibm.com/channel/23887899/playlist/651457/video/133486086

Reply:
Hi Phil,
That's a good point, we actually opened this issue last time you pointed this out: https://github.ibm.com/IAM/AM-issues/issues/2197

We'll see if we can get this put in the queue out of the backlog sometime soon.